Added hacky jQuery to smooth-scroll horizontally

sampepose opened the pull request:
I'm not sure how much "free scroll" makes sense for horizontal scrolling. There's really nothing interesting for them to see halfway between two pages, so we may as well limit them to one of the pages. This change makes the scrolling smoother and limits the view to a whole screen.

![vh](https://cloud.githubusercontent.com/assets/1855931/17612075/2cf687ae-6013-11e6-841a-77d9477d0457.gif)

sampepose commented:
cc @melvinlu @raymondethan

melvinlu commented:
@sampepose I like the idea a lot.

There are some issues with this version of the commit which may need addressing, though:

* Scrolling with a physical mousewheel takes more than one scroll for it to work (after investigating, this only occurs if you try and continuously scroll. If there is a delay in scrolling, it seems to work as intended)
* Scrolling with a trackpad and letting go early makes the page keep scrolling in whatever direction it was going in
* The HTML element should explicitly define the `height` of the page as 100% in order to correctly render the background gradient

Let me know what you think.

sampepose commented:
I see what you mean with the delay with the physical mousewheel. The delay is there on purpose as a way of avoiding the trackpad inertia problem. I can only reproduce the intertia problem if I really try on purpose.

The refreshing thing is a bug.

I see now the gradient is messed up :)

I think if we add arrow buttons somewhere on the screen that they can also click, it should be good enough. We can have the whole team try it out and fiddle with it then.

sampepose commented:
Fixed the gradient, fixed the refresh thing, added arrow buttons.

Tested on chrome and firefox with touchpad and mouse on MBP.

![vh2](https://cloud.githubusercontent.com/assets/1855931/17719641/cb0b67b2-63e1-11e6-94a8-4eda4aaff1d1.gif)

melvinlu commented:
Looks great!

I can replicate the inertia problem fairly easily, but that might be because I'm testing it on a Windows laptop. I don't think it should be that big of an issue anyways.

The refreshing bug & gradient issue changes look fixed to me!

Before this is merged, can you include a link to the fontawesome stylesheet in `index.html`? (otherwise the arrows show up as black boxes like I showed you over messenger)

```<link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.3/css/font-awesome.min.css">```

Other than that, looks great to me. (if it's possible, or maybe in the future, could we add event listeners to scroll using the arrow keys? This seems pretty intuitive from the user's end)

sampepose commented:
hmm I don't think it needs that since I include it in the JS. I wiped the repo and recloned and it still works for me. Not sure why you see a black box. Can anyone else repro?

This is what I'm using: https://gist.github.com/Turbo87/e8e941e68308d3b40ef6
